lib/psulib_traject/call_number.rb

```diff
@@ -32,11 +34,18 @@ def reverse_shelfkey
       shelf_key.reverse
     end
 
+    def not_browsable?
+      return true unless lc? || dewey?
+
+      forward_shelfkey.nil? && reverse_shelfkey.nil?
+    end
+
     def keymap
       {
-        'call_number' => value,
-        'forward_key' => forward_shelfkey,
-        'reverse_key' => reverse_shelfkey
+        call_number: value,
+        classification: classification,
```

Comment:
I used this for debugging/testing locally, not sure if we would need it for the blackcat work, might be redundant, your thought @awead? I can get rid of it if we think it is no use.

Reply:
I used that to selectively index specific call number classifications, but I guess you're using `browseable?` to determine which ones get indexed? If so, the classification parameter can probably go away.